server/data/prisonApi.ts

```diff
@@ -8,14 +8,60 @@ interface CaseLoad {
   type: string
 }
 
+// Possibly not an exhaustive list
```

Comment:
Don't think this is necessary, only interested in WING

Reply:
Yeah, I was tempted to change the `locationType` field type to string as well. I'll change as it's simpler👌